ctlearn/default_models/single_cnn.py

```diff
@@ -8,7 +8,10 @@ def single_cnn_model(data, model_params):
 
     # Load neural network model
     network_input_img = tf.keras.Input(shape=data.img_shape, name=f"images")
-    network_input_param = tf.keras.Input(shape=data.prm_shape, name=f"parameters")
+    flag_prm = 0
+    if data.prm_shape != None:
```

Comment:
Have you tested the code also in predict mode? I think here you should be also checking if we are in training mode, i.e. `if data.prm_shape != None and data.mode=="train":`

Reply:
ah yes I forgot to add that. I will update this

Reply:
done